content/en/role-based-access-control.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+The method of regulating systems and resources access based on individual users' roles within an organization.
+
+## Problem it addresses
+
+To maintain the security of an organization, it becomes increasingly important to limit unnecessary permissions based on employee roles.
```

Comment:
This is not describing the problem. It's a statement of why RBAC is necessary. This section should focus on what would a world without RBAC look like? Ideally, we shouldn't even mention the solution, just focus on the problem.

Reply:
Thanks for the feedback Cath, I'll update it now.